L1Trigger/Phase2L1ParticleFlow/src/ParametricResolution.cc

```diff
@@ -0,0 +1 @@
+#include "L1Trigger/Phase2L1ParticleFlow/interface/ParametricResolution.h"
```

Comment:
similarly, the implementation of this class's methods could be moved to the cc file